README.md

```diff
@@ -15,4 +15,13 @@ Stuff we need help with:
   pretty lean, on purpose, apps should style this however they'd like)
 - tests (eventually)
 
+# Tests!
```

Comment:
@sprjr README test instructions. Is the more information that would be useful here?